src/store/search.js

```diff
@@ -153,8 +154,11 @@ export default {
     setView(state, value) {
       state.view = value;
       if (process.browser) {
-        sessionStorage.searchResultsView = value;
-        localStorage.searchResultsView = value;
+        const consent = window.klaro.getManager(klaroConfig(this.$i18n, this.$gtm, this.$config.gtm.id)).getConsent('searchResultsView');
+        if (consent) {
```

Comment:
this conditional can be removed once  the serachResutlsView is moved to 'essential'.